webknossos/webknossos/dataset/layer.py

```diff
@@ -634,21 +634,30 @@ def _assert_mag_does_not_exist_yet(
                 )
             )
 
+    def _get_dataset_from_align_with_other_layers(
+        self, align_with_other_layers: Union[bool, "Dataset"]
+    ) -> Optional["Dataset"]:
+        if type(align_with_other_layers) is bool:
+            return self.dataset if align_with_other_layers else None
+        else:
+            return align_with_other_layers
+
     def downsample(
         self,
         from_mag: Optional[Mag] = None,
-        max_mag: Optional[Mag] = None,
+        coarsest_mag: Optional[Mag] = None,
         interpolation_mode: str = "default",
         compress: bool = True,
         sampling_mode: Union[str, SamplingModes] = SamplingModes.ANISOTROPIC,
+        align_with_other_layers: Union[bool, "Dataset"] = True,
```

Comment:
Please also mention in the changelog, that the default behavior changed (probably can be considered a fix).